public/app/dashboard/profileController.js

```diff
@@ -396,10 +462,12 @@
     var init = function() {
       // Determine user
       var userId = $stateParams.userId || (crossToolRequest && crossToolRequest.id);
+      var loadPreviousState = $stateParams.loadPreviousState !== 'false';
```

Comment:
I have satisfied myself that everything really does come out of `$stateParams` as a string.